libraries/CZ80Cpu/CZ80ACpu.h

```diff
@@ -165,6 +166,16 @@ class CZ80ACpu : public ICpu
             UINT16 *data
         );
 
+        //
+        // An implementation that has a longer wait prior
```

Comment:
The Puckman cycle was too long a wait? Just confirming. I have a change pending that adds Crazy Kong that's needed and even longer wait hold off that Puckman.

Reply:
Yeah, it was too long. I also have a Rally-X merge proposal coming up that needed it too.